config/account.go

```diff
@@ -42,6 +42,7 @@ type Account struct {
 	DefaultBidLimit         int                                         `mapstructure:"default_bid_limit" json:"default_bid_limit"`
 	BidAdjustments          *openrtb_ext.ExtRequestPrebidBidAdjustments `mapstructure:"bidadjustments" json:"bidadjustments"`
 	Privacy                 AccountPrivacy                              `mapstructure:"privacy" json:"privacy"`
+	Auction                 Auction                                     `mapstructure:"auctionprivacy" json:"auctionprivacy"`
```

Comment:
PBS-Go does not define a top level auction object. @bsardo consider if you want to introduce that in this PR. Alternatively, this config might be better suited for the `AccountPrivacy` object.

If you wish to introduce this structure, the json name `auctionprivacy` should match the field name `Auction`.

Reply:
@pm-nilesh-chate sorry for the confusion but my recollection from that meeting is that it was still up for debate. I'll come back with an answer on how to proceed here.